app/models/user.rb

```diff
@@ -0,0 +1,39 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable,
+         :confirmable
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.